cm.go

```diff
@@ -0,0 +1,70 @@
+package sachet
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"time"
+)
+
+// CMConfig is the configuration struct for CM provider
+type CMConfig struct {
+	ProductToken string `yaml:"producttoken"`
```

Comment:
for future reference: "producttoken" with no underscores is what CM uses in their api so better beconsistent with it instead of being consistent with other configurations. 👍